src/actions/walletAction.ts

```diff
@@ -149,3 +183,17 @@ export default async function () {
     error(err);
   }
 }
+
+// command function
+export default async function (walletCommand: string) {
```

Comment:
this is not needed. Th library commander already takes care of this if we use subcommand feature